.github/workflows/run-tests.yml

```
@@ -46,6 +46,10 @@ jobs:
 
       - uses: arduino/setup-protoc@v3
 
+      - run: GOOS=windows GOARCH=amd64 make bins && make clean-bins
+
+      - run: GOOS=darwin GOARCH=arm64 make bins && make clean-bins
+
       - run: make ci-build-misc
```

Also needs to call `clean-bins` here.
```suggestion
      - run: make clean-bins ci-build-misc
```

---

if I ran it after the previous step, why would I need to run `clean-bins` again?

---

In testing, `clean-bins` is actually not necessary before `make ci-build-misc`, I think because we generate code and move proto files around, so Make detects changes in the source files and recompiles the binary due to that.

But in case somehow that changes in the future, it's nice to still `clean-bins` before the next call to `bins`.

`make bins clean-bins` followed by `make bins` is equivalent to `make bins` followed by `make clean-bins bins`, so I'm pretty sure it's just an aesthetic choice there?

---

Agree it's mostly aesthetic. I prefer `make clean-bins ci-build-misc` because it's clear to me in this one line that `ci-build-misc` will certainly build new binaries. The other way, I need to track if/where the `clean-bins` is called (which in this case, it's very easy and clear).